content/Products/RHOBS/configuring-clients-for-observatorium.md

```diff
@@ -0,0 +1,233 @@
+# Configuring Clients for Red Hat’s Observatorium Instance
+
+- **Authors:**
+  - [`@squat`](https://github.com/squat)
+  - [`@spaparaju`](https://github.com/spaparaju)
+
+## Overview
+
+Teams that have identified a need for collecting their logs and/or metrics into a centralized service that offers querying and dashboarding may choose to send their data to Red Hat’s hosted Observatorium instance. This document details how to configure clients, such as Prometheus, to remote write data for tenants who have been onboarded to Observatorium following the team’s onboarding doc: [Onboarding a Tenant into Observatorium (internal)](https://docs.google.com/document/d/1pjM9RRvij-IgwqQMt5q798B_4k4A9Y16uT2oV9sxN3g/edit).
+
+## 0. Register Service Accounts and Configure RBAC
+
+Before configuring any clients, follow the steps in the [Observatorium Tenant Onboarding doc (internal)](https://docs.google.com/document/d/1pjM9RRvij-IgwqQMt5q798B_4k4A9Y16uT2oV9sxN3g/edit) to register the necessary service accounts and give them the required permissions on the Observatorium platform. The result of this process should be an OAuth client ID and client secret pair for each new service account. Save these credentials somewhere secure.
+
+## 1. Remote Writing Metrics from the Cluster Monitoring Stack to Observatorium
+
+This section describes the process of sending metrics collected by the Cluster Monitoring stack on an OpenShift cluster to Observatorium.
+
+### Background
+
+In order to remote write metrics from a cluster to Observatorium using the OpenShift Cluster Monitoring stack, the cluster’s Prometheus servers must be configured to authenticate and make requests to the correct URL. The OpenShift Cluster Monitoring ConfigMap exposes a user-editable field for configuring the Prometheus servers to remote write. Observatorium relies on OAuth to authenticate the incoming requests. Native [OAuth support for remote write](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#oauth2) was added in Prometheus v2.27.0. However, if you are using an older version of Prometheus that does not support OAuth, follow the instructions under [Process for Prometheus version < 2.27.0](#process-for-prometheus-version--2270) heading.
```

Comment:
Style: the newly added sections about Prometheus-native OAuth are the only parts of this doc that use "you" when instructing the reader. For consistency with the rest of the doc could this be changed?